core/src/main/java/org/axonframework/commandhandling/model/inspection/AnnotatedChildEntity.java

```diff
@@ -55,13 +58,20 @@ public AnnotatedChildEntity(EntityModel<C> entityModel,
         this.entityModel = entityModel;
         this.eventTargetResolver = eventTargetResolver;
         this.commandHandlers = new HashMap<>();
+        this.commandHandlerInterceptors = new ArrayList<>();
         if (forwardCommands) {
             entityModel.commandHandlers().forEach(
                     (commandType, childHandler) -> commandHandlers.put(
                             commandType,
                             new ChildForwardingCommandMessageHandlingMember<>(childHandler, commandTargetResolver)
                     )
             );
+
+            commandHandlerInterceptors.addAll(
```

Comment:
I don't think this is the right place to call child interceptors. This would mean child interceptors are invoked, even when the command handler is declared on the parent entity (?).
I would expect the interceptor invocation to happen in the `ChildForwardingCommandMessageHandlingMember`.